packages/vats/src/core/basic-behaviors.js

```diff
@@ -65,10 +65,10 @@ const bootMsgEx = {
  * @typedef {{adminNode: any, root: unknown}} VatInfo as from createVatByName
  * @typedef {MapStore<string, Promise<VatInfo>>} VatStore
  */
-export const makeVatsFromBundles = ({
+export const makeVatsFromBundles = async ({
   vats,
   devices,
-  produce: { vatAdminSvc, loadVat, vatStore },
+  produce: { vatAdminSvc, loadVat, loadCriticalVat, vatStore },
```

Comment:
I think we have no use for `loadVat` any more; I'm a little concerned that it's like a rake, waiting for somebody to step on it. I suppose an audit of manifests is straightforward... though not trivial.

Did you consider `loadVat.resolve(newFangledLoadVat)`? That is: replacing `loadVat` rather than adding a 2nd tool? Oh... I suppose that would prevent core-eval from using `loadVat` in the future.

Never mind.